arbiter-core/src/tests.rs

```diff
@@ -9,18 +9,19 @@ use ethers::{
 };
 
 use crate::{
-    agent::{tests::*, *},
+    agent::{*},
     bindings::arbiter_token::*,
-    environment::{tests::*, *},
-    manager::{tests::*, *},
-    middleware::{tests::*, *},
+    environment::{*},
+    middleware::{*},
 };
 
 const TEST_ARG_NAME: &str = "ArbiterToken";
 const TEST_ARG_SYMBOL: &str = "ARBT";
 const TEST_ARG_DECIMALS: u8 = 18;
 const TEST_MINT_AMOUNT: u128 = 1;
 const TEST_MINT_TO: &str = "0xf7e93cc543d97af6632c9b8864417379dba4bf15";
+const TEST_ENV_LABEL: &str = "test_env";
+const TEST_AGENT_NAME: &str = "test_agent";
```

Comment:
These were already sourced from `environment::tests` and `agent::tests` btw. I don't think they needed to be added here at all. The analyzer errors you're seeing are just because of the `#[cfg(test)]` that is in those modules making it so that isn't loaded in your development environment. If you try running the tests, you'll see they pass just fine.